beacon-chain/rpc/service.go

```diff
@@ -135,6 +142,7 @@ type Config struct {
 	BlobStorage                   *filesystem.BlobStorage
 	TrackedValidatorsCache        *cache.TrackedValidatorsCache
 	PayloadIDCache                *cache.PayloadIDCache
+	ProposerSettings              *validatorpb.ProposerSettingsPayload
```

Comment:
not happy using proto from validator client in beacon node, want to move this outside of validator client protobuf

Reply:
moved it out